Makefile.mule

```diff
@@ -0,0 +1,31 @@
+# This file is imported into go-algorand/Makefile.
+#
+.PHONY: ci-clean ci-deps ci-setup ci-build package package-deb package-rpm package-$(OS_TYPE)-$(ARCH) package-deb-$(OS_TYPE)-$(ARCH) package-rpm-$(OS_TYPE)-$(ARCH)
```

Comment:
Since this Makefile is being included in the main Makefile, will defining `.PHONY` here collide with the one in the main Makefile?